node/core/core.go

```diff
@@ -0,0 +1,38 @@
+package core
+
+import (
+	"go.uber.org/fx"
+
+	"github.com/celestiaorg/celestia-node/core"
+)
+
+// Config combines all configuration fields for Core subsystem.
```

Comment:
```suggestion
// Config combines all configuration fields for either embedding a Celestia Core process
// or dialling a remote Core endpoint.
```

Reply:
There will be other configuration fields for the Core subsystem, besides only embedding or dialing remotely, so I am not accepting the suggestion.

Reply:
Ok, that makes sense. How this would sound for you if I only remove 'subsystem'.

Reply:
```suggestion
// Config combines all configuration fields for communicating with a Core node.
```